<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>funcwindow</class>
 <widget class="QWidget" name="funcwindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>463</width>
    <height>381</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QListWidget" name="math_list">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>70</y>
     <width>401</width>
     <height>221</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="text_label">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>30</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>选择函数：</string>
   </property>
  </widget>
  <widget class="QLabel" name="interact_lab">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>330</y>
     <width>391</width>
     <height>19</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="correct_btn">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>320</y>
     <width>92</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>确定</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
